docs/dev_install.rst

```diff
@@ -132,22 +146,43 @@ Setting up Taichi for development
     #   cmake .. -DCUDA_VERSION=10.0 -DTI_WITH_CUDA:BOOL=True
     make -j 8
 
-- Add the following script to your ``~/.bashrc``:
+- Execute ``python3 -m taichi test`` to run all the tests. It may take up to 5 minutes to run all tests.
+- Check out ``examples`` for runnable examples. Run them with ``python3``.
 
-  .. code-block:: bash
 
-    export TAICHI_REPO_DIR=/home/XXX/taichi  # Path to your taichi repository
-    export PYTHONPATH=$TAICHI_REPO_DIR/python/:$PYTHONPATH
-    export PATH=$TAICHI_REPO_DIR/bin/:$PATH
-    # export PATH=/opt/llvm/bin:$PATH # Uncomment if your llvm-8 or clang-8 is in /opt
+Developer's Troubleshooting
+---------------------------
 
-- Execute ``source ~/.bashrc`` to reload shell config.
-- Execute ``python3 -m taichi test`` to run all the tests. It may take up to 5 minutes to run all tests.
-- Check out ``examples`` for runnable examples. Run them with ``python3``.
+- If ``make`` failed to compile and reporting ``fatal error: 'spdlog/XXX.h' file not found``,
+  please try run ``git submodule init --update --recursive --depth=1``.
+
+- If ``cmake`` failed and reporting ``clang and MSVC are the only supported compilers for Taichi compiler development``,
+  please make sure you have installed clang with version >= 7, and build with ``cmake -DCMAKE_CXX_COMPILER=clang``.
 
+- If importing Taichi cause ``FileNotFoundError: [Errno 2] No such file or directory: '/root/taichi/python/taichi/core/../lib/taichi_core.so' -> '/root/taichi/python/taichi/core/../lib/libtaichi_core.so'``, please:
+
+  * On Windows, add a environment variable called ``TAICHI_REPO_DIR`` with value ``C:\\path\\to\\taichi`` (your Taichi repo path), this may be either done in your IDE configuation panel or system settings.
+  * On Linux / OS X, add ``export TAICHI_REPO_DIR=/home/XXX/taichi`` to your ``~/.bashrc`` and restart ``bash``.
```

Comment:
Maybe we should just refer the readers to the environment variable settings instead of repeating ourselves?